spec/generate_field_yaml/generate_field_yaml_spec.rb

```diff
@@ -0,0 +1,180 @@
+require 'marc_cleanup'
+require 'byebug'
+
```

Comment:
I needed to add `require 'rspec'` here

Reply:
I don't know why you would have this problem. None of the spec files have a `require 'rspec'` line.